doc/rados/operations/health-checks.rst

```diff
@@ -396,6 +396,22 @@ paired with *PG_DAMAGED* (see above).
 
 See :doc:`pg-repair` for more information.
 
+LARGE_OMAP_OBJECTS
+__________________
+
+One or more pools contain large omap objects as determined by
+osd_deep_scrub_large_omap_object_key_threshold (threshold for number of keys to
+determine a large omap object) or
+osd_deep_scrub_large_omap_object_value_sum_threshold (the threshold for summed
+size (bytes) of all key values to determine a large omap object) or both. More
+information on the object name, key count, and size in bytes can be found by
+searching the cluster log for 'Large omap object found'.
+
```

Comment:
I think we need a quick discussion of why this might be happening. "Usually large omap objects are caused by RGW bucket index objects that do not have automatic resharding enabled.  Please see :ref:`something` for more information." or similar?

Reply:
@mattbenjamin ?